services/subscriber/config.go

```diff
@@ -17,19 +20,41 @@ type Config struct {
 	Enabled bool `toml:"enabled"`
 
 	HTTPTimeout toml.Duration `toml:"http-timeout"`
+
+	// InsecureSkipVerify gets passed to the http client, if true, it will
+	// skip https certificate verification. Defaults to false
+	InsecureSkipVerify bool `toml:"insecure-skip-verify"`
+
+	// configure the path to the PEM encoded CA certs file. If the
+	// empty string, the default system certs will be used
+	CaCerts string `toml:"ca-certs"`
 }
 
 // NewConfig returns a new instance of a subscriber config.
 func NewConfig() Config {
 	return Config{
-		Enabled:     true,
-		HTTPTimeout: toml.Duration(DefaultHTTPTimeout),
+		Enabled:            true,
+		HTTPTimeout:        toml.Duration(DefaultHTTPTimeout),
+		InsecureSkipVerify: false,
+		CaCerts:            "",
 	}
 }
 
 func (c Config) Validate() error {
 	if c.HTTPTimeout <= 0 {
 		return errors.New("http-timeout must be greater than 0")
 	}
+	if c.CaCerts != "" && !fileExists(c.CaCerts) {
+		abspath, err := filepath.Abs(c.CaCerts)
+		if err != nil {
+			return err
```

Comment:
This error should probably return something like:

```
fmt.Errorf("ca-certs file %s does not exist", c.CaCerts)
```

that way the user gets a useful error relating to the miss configuration of the CaCerts, instead of an error about not being able to determine the abspath.